cardano-git-rev/cardano-git-rev.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 
-name:                   cardano-config
+name:                   cardano-git-rev
```

Comment:
I think I would just have called it `cardano-gitrev` .

Reply:
@tmcgilchrist Does it have to be a separate package or can it be a module?

Reply:
It's easier to keep this as a small independent package since it's referred to in a number of places. Finding the root of all those to place it was used and putting the code there ended up being the worse option.